homeassistant/config_entries.py

```diff
@@ -685,6 +681,18 @@ def async_start_reauth(
             )
         )
 
+    @callback
+    def async_get_active_reauth_flows(
+        self, hass: HomeAssistant
+    ) -> Generator[FlowResult, None, None]:
```

Comment:
Thinking about this a bit more, maybe we should make this accept a set of source types `async_get_active_flows(hass, {SOURCE_REAUTH})` so we can use it for other types as well